openmetadata-service/src/main/java/org/openmetadata/service/resources/services/metadata/MetadataServiceResource.java

```diff
@@ -93,23 +94,20 @@ private void registerMetadataServices(OpenMetadataApplicationConfig config) {
                 .withElasticsSearch(getElasticSearchConnectionSink(config.getElasticSearchConfiguration()));
         MetadataConnection metadataConnection = new MetadataConnection().withConfig(openMetadataServerConnection);
         List<MetadataService> servicesList = dao.getEntitiesFromSeedData(".*json/data/metadataService/.*\\.json$");
-        servicesList.forEach(
-            (service) -> {
-              try {
-                // populate values for the Metadata Service
-                service.setConnection(metadataConnection);
-                service.setAllowServiceCreation(false);
-                dao.initializeEntity(service);
-              } catch (IOException e) {
-                LOG.error(
-                    "[MetadataService] Failed to initialize a Metadata Service {}", service.getFullyQualifiedName(), e);
-              }
-            });
+        if (servicesList.size() == 1) {
+          MetadataService service = servicesList.get(0);
+          service.setFullyQualifiedName(service.getName());
+          service.setConnection(metadataConnection);
+          service.setAllowServiceCreation(false);
+          dao.initializeEntity(service);
+        } else {
+          throw new RuntimeException("Only one Openmetadata Service can be initialized from the Data.");
```

Comment:
@mohityadav766 bit confused by this. 
We should've system flag for the OpenMetadata service that indicates that this config/service refers to the live system.
in upcoming releases we should've ability to add other OpenMetadata service to import data or export

Reply:
@harshach this is the OpenMetadata Service creation , currently we just have one service available , if we want we can add more in the bootstrap later and intilialize as per the list , but for now we just have one system service , so this check is to ensure only one is there